scl/logscale/logscale.conf

```diff
@@ -0,0 +1,77 @@
+#############################################################################
+# Copyright (c) 2023 Attila Szakacs
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+# Batching best practices: https://library.humio.com/falcon-logscale/api-ingest.html#api-ingest-best-practices
+
+@requires json-plugin
+
+
+# https://library.humio.com/falcon-logscale/api-ingest.html#api-ingest-structured-data
+block destination logscale(
+  url()
+  token()
+
+  rawstring("${MESSAGE}")
+  timestamp("${S_ISODATE}")
+  timezone("")
+  attributes("--scope rfc5424 --exclude MESSAGE --exclude DATE --leave-initial-dot")
+
+  batch_lines(5000)
+  batch_bytes(4096kB)
+  batch_timeout(5)
+  workers(8)
+  timeout(10)
+
+  content_type("application/json")
+  extra_headers("")
+  use_system_cert_store(yes)
+  ...)
+{
+
+@requires http "The logscale() driver depends on the syslog-ng http module, please install the syslog-ng-mod-http (Debian & derivatives) or the syslog-ng-http (RHEL & co) package"
+
+  http(
+    url("`url`/api/v1/ingest/humio-structured")
+    headers(
+      "Authorization: Bearer `token`"
+      "Content-Type: `content_type`"
+      "Connection: keep-alive"
```

Comment:
Keep-alive is managed by curl (and it is on by default in HTTP/1.1), so I don't think we need this here.

Reply:
Thanks, removed.